tests/test_azure_batch_executor.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import os
+
+sys.path.insert(0, os.path.dirname(__file__))
+
+from common import *
+
+
+def test_az_batch_executor():
+    bau = os.getenv("AZ_BATCH_ACCOUNT_URL")
```

Comment:
I am unsure how this should look like and where I will find it. Can you also put this info in the section of the docs? For now, I've set it to `https://snakemakebatch.eastus.batch.azure.com`.